src/parser.js

```diff
@@ -1,12 +1,41 @@
 "use strict";
 
+const codeFrame = require("babel-code-frame");
```

Comment:
since you are moving this, could you do an inline require, only require it when it's actually used.